cypress/integration/Input/Can_set_a_value/index.js

```diff
@@ -0,0 +1,19 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a Input with no text is rendered', () => {
+    cy.visitStory('Input', 'Default')
+    cy.verifyFormValue('agree', undefined)
+})
+
+When('the user types something', () => {
+    cy.wrap('user input').as('userInput')
+    cy.get('@userInput').then(userInput => {
```

Comment:
So we're storing the `'user input'` string, so that we can pass it to the `Then` step.

To be honest I'd prefer just using a simple string both times. I don't really see much benefit in relying on the `'@userInput'` string to refer to the value, over using the `'user input'` string directly.